hack/check-requisites.sh

```diff
@@ -43,16 +49,46 @@ function check_oneof {
     fi
 }
 
+check_go_version() {
+    local major=$(go version | sed -r "s|.* go([1-9]).[0-9]*[0-9.]* .*|\1|")
+    local minor=$(go version | sed -r "s|.* go[1-9].([0-9]*)[0-9.]* .*|\1|")
+
+    printf "Checking for go >= 1.$MIN_GO_VERSION... "
+    if [[ "$major" -gt 1 ]] || [[ "$minor" -ge $MIN_GO_VERSION ]]; then
+        printf "${green}ok${reset} ($major.$minor)"
+    else
+        printf "${red}ko${reset} ($major.$minor)"
+        all_found=false
+    fi
+    printf "\n"
+}
+
+check_kubectl_version() {
+    local major=$(kubectl --client=true version | grep -Eo 'Major:"[0-9]*' | grep -o '[0-9]*')
+    local minor=$(kubectl --client=true version | grep -Eo 'Minor:"[0-9]*' | grep -o '[0-9]*')
```

Comment:
This does not work for me on macOS. I think it has to be something like: 
`kubectl --client=true version | grep -Eo 'Major:"[0-9]*' | grep -Eo '[0-9]+'`